rfcs/038-sierra-series/README.md

```diff
@@ -0,0 +1,432 @@
+    "referenceNumber": "PP/DSM/3/1",
+    "totalParts": 0,
+    "totalDescendentParts": 0,
+    "type": "Work"
+  },
+  {
+    "id": "a8hbc266",
+    "title": "Articles on Dietetics",
+    "referenceNumber": "PP/DSM/3/2",
+    "totalParts": 0,
+    "totalDescendentParts": 0,
+    "type": "Work"
+  },
+  {
+    "id": "qhvzeece",
+    "title": "Nutrition Newsletters, Conference Literature etc",
+    "referenceNumber": "PP/DSM/3/3",
+    "totalParts": 0,
+    "totalDescendentParts": 0,
+    "type": "Work"
+  },
+  {
+    "id": "cpp2xh99",
+    "title": "Articles and Papers on Nutrition",
+    "referenceNumber": "PP/DSM/3/4",
+    "totalParts": 0,
+    "totalDescendentParts": 0,
+    "type": "Work"
+  },
+  {
+    "id": "ur5sakg9",
+    "title": "\"Nutrition in Britain in the twentieth century\" - PhD thesis",
+    "referenceNumber": "PP/DSM/3/5",
+    "totalParts": 0,
+    "totalDescendentParts": 0,
+    "type": "Work"
+  },
+  {
+    "id": "k4vp98bp",
+    "title": "Miscellaneous Items",
+    "referenceNumber": "PP/DSM/3/6",
+    "totalParts": 0,
+    "totalDescendentParts": 0,
+    "type": "Work"
+  }
+],
+"partOf": [
+  {
+    "id": "hb9hqd8p",
+    "title": "McLaren, Dr Donald S",
+    "referenceNumber": "PP/DSM",
+    "partOf": [],
+    "totalParts": 3,
+    "totalDescendentParts": 9,
+    "type": "Collection"
+  }
+],
+```
+For archive collections, the website renders these as a collapsible hierarchy: 
+
+![](archives.png)
+
+Given the size of many series and Host Item Entry that would not be practical in this case, so the proposal 
+is for the frontend to render differently the information in `partOf` based on the `type` property of the work. 
+
+In the case of archive works, the frontend should render the hierarchy as in the example above.
+
+In the case of Sierra works, the frontend should provide a link to a search page where the user can paginate through 
+all works linked by the same Series or Host Item Entry. 
+To do this the Catalogue API should provide a filter that matches exactly on the contents of `partOf` along the lines of
+`/works?partOf.title=Morphogenesis of the vertebrate brain`.
+
+As series and Host Item Entries can have volumes, the proposal is to represent them by creating one record for the Series or
+Host Item Entry and one for the volume with a nested `partOf` property (see example below).
+
+Additionally, if a series or Host Item Entry has an identifier (such as ISSN os ISBn or other) the proposal is 
+to mint a canonical identifier in the pipeline and to expose that identifier in the API. 
+Finally, the proposal is to model a series statement or a Host Item Entry with a type `Work`.
+
+### Examples
+ 
+- [1074785](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1074785?lang=eng&suite=cobalt&marcData=Y) 
+This is an example of a series statement in 490 + 830. In this example the information contained in 830 and 490 is 
+  exactly the same, so the result is just one statement. 
+```yaml
+"partOf": [
```

Comment:
The other option if we want to include the series both with and without the volume is to flatten them into a list, rather than trying to make it a hierarchy. It's kind of a truer representation of what we're doing with the data and might also be easier to represent on the front end, given the duplication of the whole title.

Reply:
I still think it might be better to just treat as a single partOf though initially, ie a single work that includes the volume number. Then iterate based on feedback.